<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportToPdf</class>
 <widget class="QDialog" name="ExportToPdf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Print to PDF</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="fileNameBox">
     <property name="title">
      <string>File name</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="foutFleLabel">
        <property name="text">
         <string>Output file:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="outFileEdit"/>
      </item>
      <item>
       <widget class="QToolButton" name="outFileSelectBtn">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="metaInfoBox">
     <property name="title">
      <string>Meta info</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="authorLabel">
        <property name="text">
         <string>Author:</string>
        </property>
        <property name="buddy">
         <cstring>authorEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="authorEdit">
        <property name="toolTip">
         <string> The name of the person who created the document.</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="titleLabel">
        <property name="text">
         <string>Title:</string>
        </property>
        <property name="buddy">
         <cstring>titleEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="titleEdit">
        <property name="toolTip">
         <string> The document's title.</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="subjectLabel">
        <property name="text">
         <string>Subject:</string>
        </property>
        <property name="buddy">
         <cstring>titleEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="subjectEdit">
        <property name="toolTip">
         <string> The subject of the document.</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="keywordsLabel">
        <property name="toolTip">
         <string> Keywords associated with the document.</string>
        </property>
        <property name="text">
         <string>Keywords:</string>
        </property>
        <property name="buddy">
         <cstring>titleEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="keywordsEdit">
        <property name="toolTip">
         <string> Keywords associated with the document.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>outFileEdit</tabstop>
  <tabstop>outFileSelectBtn</tabstop>
  <tabstop>authorEdit</tabstop>
  <tabstop>titleEdit</tabstop>
  <tabstop>subjectEdit</tabstop>
  <tabstop>keywordsEdit</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ExportToPdf</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ExportToPdf</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
